app/services/clean_users_service.rb

```diff
@@ -0,0 +1,14 @@
+class CleanUsersService
+  def self.call
+    count = User.where(last_sign_in_at: nil).
+            where('created_at < ?', 14.days.ago).
+            delete_all
+
+    return unless count.positive?
+
+    accounts = 'account'.pluralize(count)
+    Rails.logger.info("Deleted #{count} unauthenticated #{accounts}")
```

Comment:
I think worrying about pluralizing is nice for UIs but not as useful for semi-structured logs?

What if the message was a little more machine-readable, then we could easily search/grep for `count=`

```
Rails.logger.info("Deleted unauthenticated accounts count=#{count}")
```

Reply:
Got it.